---
uint8 waypoint_mission_count
uint8 hotpoint_mission_count